src/app/helpers/certHelper.js

```diff
@@ -0,0 +1,90 @@
+const { HTTPService } = require('@project-sunbird/ext-framework-server/services');
+const envHelper = require('./environmentVariablesHelper.js')
+const certRegURL = envHelper.LEARNER_URL
+const logger = require('sb_logger_util_v2')
+
+var certRegServiceApi = {
+    searchCertificate : 'certreg/v1/certs/search',
+    getCourse: 'content/v1/read',
+  };
+
+// To get Certificate details and course details
+  const getUserCertificates = async (data, courseId) =>  {
+    logger.debug({msg: 'getUserCertificates() is called'});
+    const userData = _.get(data, 'result.response.content[0]');
+    const options = {
+      request: {
+        _source: [],
+        query: {
+          bool: {
+            must: [
+              {
+                match_phrase: {
+                  'recipient.id': _.get(userData, 'identifier'),
+                },
+              },
```

Comment:
why do we need to add these many commas (,) ? we only have one match_phrase, right ?